src/components/EmojiPicker/EmojiPickerMenu/index.native.tsx

```diff
@@ -58,11 +55,11 @@ function EmojiPickerMenu({onEmojiSelected, activeEmoji}) {
             return;
         }
 
-        setFilteredEmojis(newFilteredEmojiList);
+        setFilteredEmojis(newFilteredEmojiList as EmojiUtils.EmojiPickerList);
```

Comment:
instead of using here assertion put comment with `@ts-expect-error` with explenation that is needed till `useEmojiPickerMenu` will be migrated to TS in that way when `useEmojiPickerMenu` will be migrated we will get ts error here that this comment is not needed anymore and we will have everything properly typed